distribution/openrct2.d.ts

```diff
@@ -510,6 +510,10 @@ declare global {
         nausea: number;
     }
 
+    interface GuestGenerationArgs {
+        readonly id: number;
+    }
```

Comment:
Not particularly important, but naturally this would go under `ActionLocationArgs`, if it were to be consistent with the order of the subscribe overloads.

Reply:
I'll adjust it